samples/net/sockets/big_http_download/src/big_http_download.c

```diff
@@ -195,7 +208,8 @@ void download(struct addrinfo *ai, bool is_tls)
 		mbedtls_md_update(&hash_ctx, response, len);
 
 		cur_bytes += len;
-		printf("%u bytes\r", cur_bytes);
+		printf("Load progress: %u Bytes; %u KiB; %u MiB\r",
```

Comment:
"Load progress" sounds somewhat confusing, unclear which "load" is meant. What happens here is *download*. So, please either make it "%u Bytes; %u KiB; %u MiB\r" to save on the output size (note that on high networking speed, UART output may affect the throughput). Or otherwise, make it unambiguous with "Download progress: %u Bytes; %u KiB; %u MiB\r".

Reply:
"Download progress" makes more sense.  You're right.
I absolutely agree, this printf will slow down the speed.
I thought to bring this printf into separted tread. What do you think about?

Reply:
> I absolutely agree, this printf will slow down the speed.

Then you understand why the original message was just "%u bytes\r" and nothing else ;-).

> I thought to bring this printf into separted tread. What do you think about?

Please don't. As I already mentioned, this should be a **simple** sample, it should not be an industrial-grade networking throughput testing tool. If you're interested in having better means of speed testing, that should be a separate application. And we actually have such, it's https://github.com/zephyrproject-rtos/zephyr/tree/master/samples/net/zperf . I'd suggest you actually pay more attention to it instead - it needs more people to use it, and certainly some improvements too.

Reply:
Ok. Understand it. Thank you.